students/Vincent_Aquila/session03/list_lab.py

```diff
@@ -0,0 +1,185 @@
+"""
+response = input("Please select one other fruit you want added to this list. > ")
+new_fruit = response.title()
+"""
+The purpose of .title() is to ensure the first letter of the word is capitalized.
+Later in the program, in a for loop, the loop searches for words, and those words
+begin with a capital letter.  If a user entered a word, where the first letters
+is not capitalized, then the program would not work properly.  .title() ensures
+the program works correctly.
+"""
+#print(new_fruit) #test point to see the new fruit that was requested
+
+list.append(new_fruit)
+print(list) #this prints the amended list with the new fruit added to the end
+
+response = input("Please pick a number between 0 and 4. > ")
+number = response
+print(number)
+#note, further work could be done - when entering numbers outside 0-4
+
+for j, item in enumerate(list): #generates two columns: numbers and fruits
+    print(j, item)
+print(list)
+
+#add second fruit to the beginning of the list using +
+list.insert(0, "Strawberry")
+print(list)
+"""
+I learned a lot here - tried to use:
+    l = (str(k)[1:-1])
+which removes brackets from the list in the terminal display - but - after
+stripping brackets from a list and then employing an insert method -
+it won't work - throws an attribute error.  Insert only works with a bracketed
+list.  Unable to use the + sign to add the second fruit to the beginning of the
+list.
+"""
+
+#add a thrid fruit to the beginning of the list using insert()
+list.insert(0, "Kiwi")
+print(list)
+
+#display all the friuts that begin with "P" using a for loop
+for item in list:
+	if "P" in item:
+		print(item)
+print("---End of Series 1 Exercises---")
+#end of Series 1
+
+
+
+#Series 2
+"""
+Using the list created in series 1 above:
+-Display the list.
+-Remove the last fruit from the list.
+-Display the list.
+-Ask the user for a fruit to delete, find it and delete it.
+(Bonus: Multiply the list times two. Keep asking until a match is found. Once
+found, delete all occurrences.)
+"""
+
+print(list)
+list2 = list
+#remove the last fruit from the list and display the new list
+list2.pop()
+print(list2)
+
+#ask the user for a fruit to delete and delete it
+response = input("Select a fruit to delete from the list. > ")
+response = response.title()
+list2.remove(response)
+print(list2)
+
+double_list = list2 * 2
+print(double_list)
+print("---End of Series 2 Exercises---")
+#end of Series 2
+
+
+
+#Series 3
+"""
+Again, using the list from series 1:
+-Ask the user for input displaying a line like “Do you like apples?” for each
+fruit in the list (making the fruit all lowercase).
+-For each “no”, delete that fruit from the list.
+-For any answer that is not “yes” or “no”, prompt the user to answer with one of
+those two values (a while loop is good here)
+-Display the list.
+"""
+
+list3 = ["Apples", "Pears", "Oranges", "Peaches"] #original list
+response_apples = input("Do you like apples?  Please respond with yes or no > ")
```

Comment:
you do not want to hard-code these particular fruits!

you should be able to loop though your list.